src/core/api_client/tests/factories/client.factory.ts

```
@@ -0,0 +1,29 @@
+      authConfig.clientId,
+    );
+    return authClient;
+  }
+
+  static createApiClient(authClient: OidcSimpleClient): ApiClient {
+    const apiClient = new ApiClient({});
+    apiClient.init({ baseUrl: AuthFactory.getApiUrl() });
+    apiClient.authService = authClient;
+    return apiClient;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)